civic_europe/templates/formtools/wizard/wizard_form.html

```diff
@@ -5,6 +5,7 @@
 
 {% block content %}
 <div class="wizard-page">
+    <div class="container-narrow wizard-warning-text text-center">{% trans "Please note: Your application will NOT be saved until you hit the “Submit” button at the very end of the application form. Please save an offline version of your answers to avoid data lost" %}</div>
```

Comment:
I inserted a 
`{% if wizard.steps.current == wizard.steps.first %}`
because I think it is only supposed to show on the first page, right?

Reply:
looking at the story, it's written a little weirdly so it's not clear?! but as you have the notes i'll go with you :)